plugins/catalog-import/src/components/ImportInfoCard/ImportInfoCard.tsx

```diff
@@ -53,46 +57,55 @@ export const ImportInfoCard = (props: ImportInfoCardProps) => {
 
   return (
     <InfoCard
-      title="Register an existing component"
+      title={t('importInfoCard.title')}
       titleTypographyProps={{ component: 'h3' }}
       deepLink={{
-        title: 'Learn more about the Software Catalog',
+        title: t('importInfoCard.deepLinkTitle'),
         link: 'https://backstage.io/docs/features/software-catalog/',
       }}
     >
       <Typography variant="body2" paragraph>
-        Enter the URL to your source code repository to add it to {appTitle}.
+        {t('importInfoCard.linkDescription', { appTitle })}
       </Typography>
       <Typography component="h4" variant="h6">
-        Link to an existing entity file
+        {t('importInfoCard.fileLinkTitle')}
       </Typography>
       <Typography variant="subtitle2" color="textSecondary" paragraph>
-        Example: <code>{exampleLocationUrl}</code>
+        {t('importInfoCard.examplePrefix')}
+        <code>{exampleLocationUrl}</code>
       </Typography>
       <Typography variant="body2" paragraph>
-        The wizard analyzes the file, previews the entities, and adds them to
-        the {appTitle} catalog.
+        {t('importInfoCard.fileLinkDescription', { appTitle })}
       </Typography>
       {hasGithubIntegration && (
         <>
           <Typography component="h4" variant="h6">
-            Link to a repository{' '}
-            <Chip label="GitHub only" variant="outlined" size="small" />
+            {t('importInfoCard.githubIntegration.title')}
+            <Chip
+              label={t('importInfoCard.githubIntegration.label')}
+              variant="outlined"
+              size="small"
+              style={{ marginLeft: 8, marginBottom: 0 }}
+            />
           </Typography>
           <Typography variant="subtitle2" color="textSecondary" paragraph>
-            Example: <code>{exampleRepositoryUrl}</code>
+            {t('importInfoCard.examplePrefix')}
+            <code>{exampleRepositoryUrl}</code>
           </Typography>
           <Typography variant="body2" paragraph>
-            The wizard discovers all <code>{catalogFilename}</code> files in the
-            repository, previews the entities, and adds them to the {appTitle}{' '}
-            catalog.
+            {t('importInfoCard.githubIntegration.descriptionPrefix')}
```

Comment:
I think this is another one of those `prefix` `suffix` things that we want to avoid here too

Reply:
Done!